pages/login.js

```diff
@@ -67,9 +68,13 @@ const Login = ({}) => {
       });
   };
 
-  const handleRequestEmailLink = () => {
-    setSentEmailLoginRequest(true);
-    //TODO: send email login request to api
+  async function handleRequestEmailLink(data) {
```

Comment:
@taylorz I'm pretty sure this function doesn't have access to `data` like the `onSubmit` function above. Do we want to just grab the email from the form field above? or should we display a different form field w/ just an email field?

Reply:
Hey, I updated this function re: our conversation in Slack!
https://github.com/keithtom/wildflower-journey-ui/pull/174